backend/business-services/egov-hrms/src/main/resources/db/migrate.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+baseurl=$DB_URL
+echo "the baseurl : $DB_URL"
+schemasetter="?currentSchema="
+schemas=$SCHEMA_NAME
+echo "the schemas : $schemas"
+for schemaname in ${schemas//,/ }
```

Shellcheck (0.10.0)

[warning] In POSIX sh, string replacement is undefined.

(SC3060)